websites/T/TomsHardware/presence.ts

```diff
@@ -0,0 +1,93 @@
+const presence = new Presence({
+  clientId: "889780989386170420"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "tomshardware"
+  };
+
+  let { pathname } = document.location;
```

Comment:
You are never reassigning the value of `pathname`, you should use `const` instead
```suggestion
  const { pathname } = document.location;
```

Reply:
CodeFactor failed when committing this.
Issue is that I am reassigning it, 2 lines below (L12 and L13), when it's a link with "/uk"